tests/Integration/AbstractTechproductsTest.php

```diff
@@ -647,6 +648,104 @@ public function testGetStatusFromAllCoreWhenNoCoreNameWasSet()
         $response = $this->client->coreAdmin($coreAdminQuery);
         $this->assertTrue($response->getWasSuccessful());
     }
+
+    public function testManagedStopwords()
+    {
+        $query = $this->client->createManagedStopwords();
+        $query->setName('english');
+        $term = 'the';
+
+        // Add stopwords
+        $add = new AddStopwords();
+        $add->setStopwords([$term]);
+        $query->setCommand($add);
+        $result = $this->client->execute($query);
```

Comment:
According to https://lucene.apache.org/solr/guide/7_5/managed-resources.html#managed-resources "you must reload the core/collection before changes become active".

Reply:
Reloading is only necessary for Solr components (e.g. ManagedSynonymGraphFilterFactory). It has nothing to with our tests. We can't just blindly reload the core/collection because it has all sorts of implications. It would be an action purposely initiated by a user.

Reply:
I agree, that we should not trigger a reload after the command automatically. But it needs to be documented that the user needs to reload the core or the collection at some point.

I just wondered if the tests will fail due to that. Right now they didn't run because of these class not found issues.